consensus/polybft/fsm_test.go

```diff
@@ -710,6 +710,69 @@ func TestFSM_ValidateCommit_Good(t *testing.T) {
 	require.NoError(t, err)
 }
 
+func TestFSM_Validate_ExitEventRootNotExpected(t *testing.T) {
+	t.Parallel()
+
+	const (
+		accountsCount     = 5
+		parentBlockNumber = 25
+		signaturesCount   = 3
+	)
+
+	validators := validator.NewTestValidators(t, accountsCount)
+	parentExtra := createTestExtraObject(validators.GetPublicIdentities(), validator.AccountSet{}, 4, signaturesCount, signaturesCount)
+	parentExtra.Validators = nil
```

Comment:
Why it is important to set this to nil? I don't get it at the first glance.

Same here: https://github.com/0xPolygon/polygon-edge/pull/1638/files#diff-bf0766970c7dcf6e2f9546a5f524d03a9e2cefcb411b6838d3b74a3a500c15eeR736